presto-hive/src/main/java/com/facebook/presto/hive/clustering/Interval.java

```diff
@@ -0,0 +1,26 @@
+package com.facebook.presto.hive.clustering;
+
+import com.facebook.presto.hive.HiveType;
+
+import java.util.List;
+
+public class Interval
```

Comment:
We can just use a single `Block` to represent this. Or `Domain` probably is another class that is closer to this. Let's chat in person.